Content.Server/GameTicking/GameTicker.GamePreset.cs

```diff
@@ -42,27 +44,38 @@ void FailedPresetRestart()
 
             if (_configurationManager.GetCVar(CCVars.GameLobbyFallbackEnabled))
             {
-                ClearGameRules();
-                SetGamePreset(_configurationManager.GetCVar(CCVars.GameLobbyFallbackPreset));
-                AddGamePresetRules();
-                StartGamePresetRules();
+                var fallbackPresets = new [] { _configurationManager.GetCVar(CCVars.GameLobbyFallbackPreset) , SecondaryFallback };
+                var startFailed = true;
 
-                startAttempt.Uncancel();
-                RaiseLocalEvent(startAttempt);
-
-                _chatManager.DispatchServerAnnouncement(
-                    Loc.GetString("game-ticker-start-round-cannot-start-game-mode-fallback",
-                        ("failedGameMode", presetTitle),
-                        ("fallbackMode", Loc.GetString(Preset!.ModeTitle))));
+                foreach (var preset in fallbackPresets)
+                {
+                    ClearGameRules();
+                    SetGamePreset(preset);
+                    AddGamePresetRules();
+                    StartGamePresetRules();
+
+                    startAttempt.Uncancel();
+                    RaiseLocalEvent(startAttempt);
+
+                    if (!startAttempt.Cancelled)
+                    {
+                        _chatManager.DispatchServerAnnouncement(
```

Comment:
this should probably be admin only since it gives players easy meta knowledge

Reply:
Done!